Jade_case.js

```diff
@@ -0,0 +1,117 @@
+{
+	"translatorID": "c7e07d17-cc0b-4f6c-b12a-6f3a9de0410b", // generate with `uuidgen` if needed
+	"label": "JADE Case Extractor",
```

Comment:
```suggestion
	"label": "JADE",
```

And rename the file to JADE.js.